src/core/dfn-panel.js

```diff
@@ -10,7 +10,7 @@ export const name = "core/dfn-panel";
 export async function run() {
   const css = await loadStyle();
   document.head.insertBefore(
-    hyperHTML`<style>${css}</style>`,
+    hyperHTML`<style class="removeOnSave">${css}</style>`,
```

Comment:
We want to enable dfn panel on saved documents, no?

Reply:
yes, we want to. but not yet. gotta fix echidna breakage first. will add it back with the fix in https://github.com/w3c/respec/pull/2820

Reply:
Did you get a change to see other two attempts?
https://github.com/w3c/respec/pull/2846/commits/86e8f5a88d0e7be7b00f192c610516a54d9f4e6f is good if we want a workaround on css-validator, but way too hacky.
https://github.com/w3c/respec/pull/2846/commits/fcc0c9a662e7e1a1b0181e2d287c017332c3b298 is good practice, but I don't like adding markup for decorative changes.